modules/installation-machines-bare-metal.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-machines-bare-metal-{context}"]
+= Creating the cluster
+
+To create the {product-title} cluster, you provision machines by using the Ingition config
+files that you generated with the installation program.
+
+.Prerequisites
+
+* Your machines must have internet access.
+* Create the required infrastructure for the cluster.
+* You obtained the installation program and generated the Ignition config files
+for your cluster.
+
+.Procedure
+
+. Obtain the {op-system-first} images.
+
+. Use the the ignition config files that you generated by using the installation
+program to create the required bootstrap, worker, and master nodes.
+
+. Monitor for bootstrap-complete
```

Comment:
Maybe reword this step to say that you should watch for the *bootstrap-complete* success message to confirm that the bootstrap process completed?

Reply:
This module shouldn't be in this PR, so I'm fixing this on the bare metal PR. Thanks!